modules/adding-custom-notification-banners.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * web-console/customizing-the-web-console.adoc
+
+[id="creating-custom-notification-banners_{context}"]
+= Creating custom notification banners
+
+.Procedure
+
+. From *Administration* -> *Custom Resource Definitions*, click on
+*ConsoleNotification*.
+
+. Click *YAML* and edit the file:
++
+----
+
```

Comment:
This is another one where the `golang` is helpful if you want to read the possible options (via the comments).

Reply:
But a customer would probably create a `yaml` and either `oc create -f <the-yaml>.yaml`, or would paste it into the console via the instructions you have above.

Here is a quick example of a `yaml` file for this:

```yaml
apiVersion: console.openshift.io/v1
kind: ConsoleNotification
metadata:
  name: example
spec:
  backgroundColor: '#0088ce'
  color: '#fff'
  link:
    href: 'https://www.example.com'
    text: Optional link text
  location: BannerTop
  text: This is an example notification message with an optional link.
```